src/main/java/sparksoniq/spark/udf/GroupClauseCreateColumnsUDF.java

```diff
@@ -71,9 +72,12 @@ public Row call(WrappedArray wrappedParameters) {
             // nulls and empty sequences have special grouping captured in the first grouping column
             // if non-null, non-empty-sequence value is given, the second column is used to group the input
             // indices are assigned to each value type for the first column
-            int emptySequenceGroupIndex = 1;         // by default, empty sequence is taken as first(=least)
-            int nullGroupIndex = 2;                  // null is the smallest value except empty sequence(default)
-            int valueGroupIndex = 3;                 // values are larger than null and empty sequence(default)
+            int emptySequenceGroupIndex = 1;
+            int nullGroupIndex = 2;
+            int trueGroupIndex = 3;
```

Comment:
I think I would prefer booleanTrueGroupIndex and booleanFalseGroupIndex here for clarity. What do you think?